spec/models/offer_spec.rb

```diff
@@ -123,4 +123,13 @@
     end
   end
 
+  describe "#send_thank_you_message" do
+    it 'should send thank you message to donor on offer submit' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.